### PR TITLE
Optimizations

### PR DESCRIPTION
**Pull request by kyematzen**
Lamba optimizations, variable modifications, code reductions, and removal of checks that were true before checked meaning unnecessary.

### CODE REVIEW

**Comment by Geolykt** [comment]
This is more of a code restyle than actual optimisations at first glance

File: src/main/java/de/geolykt/enchantments_plus/evt/Watcher.java [hunk 1]
```diff
@@ -275,34 +273,33 @@ public void onEnchantItem(EnchantItemEvent evt) {
...
-                            case WEIGHT:
-                                if (player.getPersistentDataContainer().has(Weight.ACTIVE, PersistentDataType.BYTE)) {
-                                    player.removePotionEffect(PotionEffectType.INCREASE_DAMAGE);
-                                    player.removePotionEffect(PotionEffectType.SLOW);
-                                    player.getPersistentDataContainer().remove(Weight.ACTIVE);
-                                }
-                                break;
-                            default:
-                                break;
+        evt.getInventory();
```
Comment (Geolykt) [inline src/main/java/de/geolykt/enchantments_plus/evt/Watcher.java:276]: Please avoid submitting code your IDE generated

File: src/main/java/de/geolykt/enchantments_plus/util/ColUtil.java [hunk 2]
```diff
@@ -31,1162 +31,1162 @@
...
-            return AbstractDyeableType.WOOL;
-        default:
-            break;
+            case BLACK_BANNER:
+                return AbstractDyeableType.BANNER;
+            case BLACK_BED:
+                return AbstractDyeableType.BED;
+            case BLACK_CARPET:
+                return AbstractDyeableType.CARPET;
+            case BLACK_CONCRETE:
```
Comment (Geolykt) [inline src/main/java/de/geolykt/enchantments_plus/util/ColUtil.java:46]: yeah. You get the idea.
If you are submitting 2.2k lines of source code changes within a file, you should review if the changes made sense

File: src/main/java/de/geolykt/enchantments_plus/util/Utilities.java [hunk 3]
```diff
@@ -222,11 +222,12 @@ public static Location getCenter(Location loc, boolean centerVertical) {
...
-            z += .5;
-        } else {
-            z += .5;
-        }
+//        if (z >= 0) {
+//            z += .5;
+//        } else {
+//            z += .5;
+//        }
+        z += .5;
```
Comment (Geolykt) [inline src/main/java/de/geolykt/enchantments_plus/util/Utilities.java:230]: smells like a bug. You should've corrected it or at least inserted a FIXME in it

**Comment by kyematzen** [comment]
I see. I'll test it again later and test in-game if I can make a few optimizations to the enchantments. 

Saw a few earlier although had no time and it'll require code changes internally.

**Review by Geolykt — COMMENTED** [review]
Thanks for the PR, however other than optimizing if statements there hasn't been a lot of optimisations in this PR, so a better name like "Improved code styling" would have been a better thing to do or (what in my opinion would have been better) do two PRs.
I'll likely merge the PR and remove non-beneficial changes (such as the SemVer breaks).

Also thanks for that one bug sniff, I'll see if that was an actual bug or just some dumb coding.

File: src/main/java/de/geolykt/enchantments_plus/Config.java [hunk 4]
```diff
@@ -33,43 +42,154 @@
...
-import de.geolykt.enchantments_plus.util.Tool;
-
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.lang.reflect.InvocationTargetException;
 import java.util.*;
 
+enum ConfigKeys {
```
Comment (Geolykt) [inline src/main/java/de/geolykt/enchantments_plus/Config.java:52]: You should put the public class first, then the non-public ones by convention (just for future's sake and so you know why I'm reverting some changes of yours)

**Pull request merged**